packages/components/addon/components/hds/avatar/index.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="hds-avatar" ...attributes data-test-avatar aria-hidden="true">
+  {{#if @src}}
+    <img src={{@src}} alt="" role="none" data-test-avatar-img />
+  {{else}}
+    <FlightIcon @name="user-circle" @size="24" />
+  {{/if}}
+</div>
```

Comment:
I considered it; but considering that I was trying to do as close of a 1:1 port as possible, _and_ we don't have an HTML validator in our build yet, _and_ this markup is similar to other markup that we have in our components, I decided to leave the divs there. Definitely open to having a team discussion and changing this across the board in a future PR, though. 👍

Reply:
Thanks for the background information. 

It would be great if we could look into improving markup in the future to try to validate. I understand we have other priorities currently though. :)